main.cpp

```diff
@@ -16,11 +16,13 @@
 
 #include "test_driver.h"
 #include "test_host.h"
+
```

Comment:
Nit: remove (I think `clang-format` might do this for you, you may need to run it manually since I'm not sure if the hook re-checks all the files on an amend).

Reply:
done